python/lsst/ctrl/stats/records/evicted.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, year, lines):
         # bytes received during the run
         self.runBytesReceived = int(self.extract(pat, lines[5], "bytes"))
 
-        pat = r"Partitionable Resources :\s+Usage\s+\Request\s+Allocated$"
+        pat = r"Partitionable Resources :\s+Usage\s+\s+Request\s+Allocated$"
```

Comment:
I find `\s+\s+` a bit confusing because it takes some thinking to work out that this is matching 2 or more spaces. Maybe `\s\s+` would be more obvious? Or even something more explicit like `\s{2,}` (which I think is Perl syntax so I haven't checked that for python).